extensions/ql-vscode/test/pure-tests/query-test.ts

```diff
@@ -10,6 +10,7 @@ import * as qsClient from '../../src/queryserver-client';
 import * as cli from '../../src/cli';
 import { ProgressReporter, Logger } from '../../src/logging';
 import { ColumnValue } from '../../src/bqrs-cli-types';
+import { FindDistributionResultKind } from '../../src/distribution';
```

Comment:
Oh I think this change introduces a test dependency on `vscode`, which `pure-tests` have to avoid.

Reply:
I'm wondering if this distinction is still worth it. If it's not easy to remove this dependency, then I may move `query-test.ts` to the integration test.

Reply:
And thanks for tracking that down. I was about to look into this myself.